documentation/docs/getting_started/installation/docker_compose/docker_compose.md

```diff
@@ -66,7 +72,7 @@ IOTA_HORNET_NODE=https://chrysalis-nodes.iota.org:443
 DATABASE_NAME=integration-services
 MONGO_INITDB_ROOT_USERNAME=root
 MONGO_INITDB_ROOT_PASSWORD=rootpassword
-DATABASE_URL=mongodb://root:rootpassword@mongo:27017
+DATABASE_URL=mongodb://root:rootpassword@localhost:27017
```

Comment:
can you check if it also works with 0.0.0.0 instead of localhost and use this if so, as I remember localhost not works in any case

Reply:
Yes 0.0.0.0 does also work for me. So lets stick with 0.0.0.0